modules/minimap2/1.0/config/default.yaml

```diff
@@ -0,0 +1,36 @@
+lcr-modules:
+    
+    minimap2:
+
+        # TODO: Update the list of available wildcards, if applicable
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_fastq: "__UPDATE__"
```

Comment:
Are any indexes required for this tool to work? Maybe we should add them here as inputs as well?

Reply:
No index files are required to run this tool